ts/hooks/useLoadMunicipality.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from "react";
+import { pipe } from "fp-ts/lib/function";
+import * as E from "fp-ts/lib/Either";
+import { useIODispatch, useIOSelector } from "../store/hooks";
+import { profileFiscalCodeSelector } from "../store/reducers/profile";
+import { CodiceCatastale } from "../types/MunicipalityCodiceCatastale";
+import { contentMunicipalityLoad } from "../store/actions/content";
+
+/**
+ * A custom hook use to load content municipality when needed
+ */
+export const useLoadMunicipality = () => {
+  const fiscalCode = useIOSelector(profileFiscalCodeSelector);
```

Comment:
What about adding a comment about the reason why you exported this hook, but actually it's not use anymore? What do you think? Or we could even remove it and if needed we can restore it just in case.

Reply:
In the old FiscalCode screen, this logic was used to load municipality information. Since it is not necessary in the new screen, I chose to extract it into a custom hook instead of deleting it. However, if you tell me it can be removed, I will do it!

Reply:
I like to leave it as you did too. But I will add a comment that explain the reason why of this hook and why actually it's not used anymore. The logic to [load the municipality is still in place](https://github.com/search?q=repo%3Apagopa%2Fio-app%20contentMunicipalityLoad.request&type=code), so I like what you did. Just document the refactoring. What do you think?

Reply:
Done, I've added a description to explain why this hook was created but isn't in use yet